motr/m0crate/crate_index.c

```diff
@@ -238,6 +243,7 @@ static void cr_time_measure_begin(struct cr_time_measure_ctx *t)
 static double cr_time_measure_elapsed_now(struct cr_time_measure_ctx *t)
 {
 	m0_time_t d = m0_time_sub(m0_time_now(), t->ts);
+	t->test_time = d;
```

Comment:
This can be 
t->ts_next = m0_time_now()
which seems to be missing.

test_time not required and can be calculated from m0_time_sub(ts_next, ts) like (last - first)